plugin/src/main/java/org/opensearch/ml/action/syncup/TransportSyncUpOnNodeAction.java

```diff
@@ -141,9 +144,13 @@ private MLSyncUpNodeResponse createSyncUpNodeResponse(MLSyncUpNodesRequest loadM
             mlModelManager.syncModelRouting(modelRoutingTable);
```

Comment:
Is it useful to add a log info here showing an import action is being taken like this "clear Routing Table"? Same for remove worker nodes, add work nodes, etc?